packages/remix-react/routeMatching.ts

```diff
@@ -1,7 +1,7 @@
 // TODO: We eventually might not want to import anything directly from `history`
 // and leverage `react-router` here instead
 import type { Location } from "history";
-import type { Params, RouteObject } from "react-router"; // TODO: export/import from react-router-dom
+import type { Params, RouteObject } from "react-router-dom";
```

Comment:
These changes aren't necessary in this PR and could be done in a separate PR, so let's revert them to keep this PR focussed

```suggestion
import type { Params, RouteObject } from "react-router"; // TODO: export/import from react-router-dom
```